diff --git a/dist/runtime/app/plugins/multi-tab.client.js b/dist/runtime/app/plugins/multi-tab.client.js
index a41f4eedb5a3a450965aebedfadfbf93b1f36ea8..84938ddc7373d6cc07b3fa7d28a06df3118c6f64 100644
--- a/dist/runtime/app/plugins/multi-tab.client.js
+++ b/dist/runtime/app/plugins/multi-tab.client.js
@@ -41,7 +41,10 @@ export default defineNuxtPlugin({
       return;
     const channel = new BroadcastChannel(CHANNEL_NAME);
     let receivedFromChannel = false;
+    let channelClosed = false;
     nuxtApp.hooks.hook("app:manifest:update", (manifest) => {
+      if (channelClosed)
+        return;
       if (receivedFromChannel) {
         receivedFromChannel = false;
         return;
@@ -57,6 +60,7 @@ export default defineNuxtPlugin({
       }
     };
     nuxtApp.hook("app:error", () => {
+      channelClosed = true;
       channel.close();
     });
   }
